tests/decap/test_subnet_decap.py

```diff
@@ -0,0 +1,226 @@
+import pytest
+import logging
+import json
+import time
+import random
+from collections import defaultdict
+
+import ptf.packet as packet
+import ptf.testutils as testutils
+from ptf.mask import Mask
+from tests.common.dualtor.dual_tor_utils import rand_selected_interface     # noqa F401
+from tests.common.fixtures.ptfhost_utils import skip_traffic_test           # noqa F401
+from tests.common.fixtures.ptfhost_utils import copy_arp_responder_py       # noqa F401
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('t0', 't1')
```

Comment:
this testcase is only running on `t0`

Reply:
Maybe we can add `dualtor`, because once we add `Verify IPinIP packet targeting at vnet route is decapsulated` to this script, it should be run on t1